package.json

```diff
@@ -29,6 +29,7 @@
     "lib": "lib"
   },
   "dependencies": {
+    "mailgun-js": "^0.7.7",
```

Comment:
I don't put appender dependencies in to package.json - not all users of log4js will need mailgun, and they should not have to have it installed. People who want to use it will have to install it as part of their own dependencies (this is similar to the requirements for the smtp appender, for example).